Examples/Detectors/MagneticField/CMakeLists.txt

```diff
@@ -12,3 +18,25 @@ target_link_libraries(
 )
 
 acts_compile_headers(ExamplesMagneticField GLOB "include/**/*.hpp")
+
+# Add Python bindings for ToroidalField if Python bindings are enabled
+if(ACTS_BUILD_EXAMPLES_PYTHON_BINDINGS)
```

Comment:
I believe this would go into https://github.com/acts-project/acts/blob/main/Python/Core/CMakeLists.txt